src/BackpackServiceProvider.php

```diff
@@ -63,6 +63,7 @@ public function boot(Router $router)
         $this->sendUsageStats();
 
         Basset::addViewPath(realpath(__DIR__.'/resources/views'));
+        Blade::component('datatable', Datatable::class);
```

Comment:
Why do we need to register the component manually? Do we not have a namespace/directory where they get registered automatically?